jerry-core/parser/js/js-parser.c

```diff
@@ -20,6 +20,8 @@
 #include "jcontext.h"
 #include "js-parser-internal.h"
 
+#ifndef CONFIG_DISABLE_PARSER_BUILTIN
```

Comment:
Sounds good, i was not certain how to call it. Let me change it,
I checked on the rest of the code and you guys don't guard the includes, that is why is there after the js-parser-internal.h